notifique/src/main/res/layout/list_item_children.xml

```diff
@@ -5,11 +5,21 @@
       android:layout_height="wrap_content"
       android:orientation="horizontal"
       >
+    <ImageView
+        android:id="@+id/icon_picture"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="start"
+        android:adjustViewBounds="true"
```

Comment:
remove?

Reply:
removed layout_gravity="start" 
needed viewbounds for sizing